frinx-uniconfig/user-guide/uniconfig-operations/uniconfig-queries/index.md

```diff
@@ -0,0 +1,232 @@
+is done only on the database side - UniConfig receives already narrowed configuration with only selected data.
+Since query is evaluated by the database, this feature is works only with already committed data (operational data).
+
+The following sequence diagram captures the whole process of RPC execution in detail.
+
+![Execution of RPC query-config](puml-RPC_query_config.svg)
+
+### RPC input fields
+
+- **topology-id**: Identifier of network-topology/topology list entry. Currently, supported topologies, under which
+  this RPC can be used, are: 'uniconfig', 'templates', 'unistore', and snapshot topologies.
+- **node-id**: Identifier of specific network-topology/node list entry which configuration is filtered using specified
+  jsonb-path-query.
+- **jsonb-path-query**: JSONB-path query used for selection and filtering of subtrees in the node configuration stored
+  in the PostgreSQL. JSONB-path must start from root "frinx-uniconfig-topology:configuration" container
+  (it is always represented by absolute path).
+
+!!!
+JSONB-path query syntax is specified by PostgreSQL. You can find detailed description of all features with examples
+on the following link (version 14):
+https://www.postgresql.org/docs/14/functions-json.html#FUNCTIONS-SQLJSON-PATH
+!!!
+
+### RPC output fields
+
+- **config**: List of selected and filtered JSON objects. Note that database may return multiple list entries,
+  if the last element in the JSONB-path is represented by list/leaf-list YANG schema node. In other cases, only
+  one or no JSON object is displayed on output based on fulfilling the filtering and selection criteria.
+
+### Example: selection of JSON object
+
+The following request demonstrated execution of simple selection query under the 'dev01' from 'uniconfig' topology.
+Response contains 1 JSON object - 'ssh' container.
+
+```bash RPC Request
+curl --location --request POST 'http://127.0.0.1:8181/rests/operations/uniconfig-query:query-config' \
+--header 'Accept: application/json' \
+--header 'Content-Type: application/json' \
+--data-raw '{
+    "input": {
+        "node-id": "dev01",
+        "topology-id": "uniconfig",
+        "jsonb-path-query": "$.\"frinx-uniconfig-topology:configuration\"
+        .\"system\"
+        .\"ssh\""
+    }
+}'
+```
+
+```json RPC Response, Status: 200
+{
+    "output": {
+        "config": [
+            {
+                "keepalive-interval": 300,
+                "max-retries": 0
+            }
+        ]
+    }
+}
+```
+
+!!!
+JSONB-path query should always start with $.\"frinx-uniconfig-topology:configuration\" pattern because 'configuration'
+represents wrapping element for all root data elements that are stored in database.
+!!!
+
+!!!
+Be aware that PostgreSQL requires escaping of special characters in the identifiers of JSON elements. For example,
+':' and '-' represents special characters. Because of this behaviour, it is always safer to put double quotes around
```

Comment:
For example, ':' and '-' ~~represents~~ **represent** special characters.

Reply:
fixed in next commit